### ISSUE TITLE
Using strict mode

### ISSUE THREAD

**Issue by sp90** [johnpapa/angular-styleguide#569]
The issue is basicly i just turn on jshint on my project and it prompts me 

Unnecessary directive "use strict".

Even though i have followed the guide and i use the exact same jshint setup as the guide - how can this be?

**Comment by zachlysobey** [comment]
Are you [wrapping all your things in IIFE's](https://github.com/johnpapa/angular-styleguide#iife)?

Maybe paste an example of the code it is complaining about?

**Comment by sp90** [comment]
Yes, i wrap all the code in iffe's :+1: 

```
(function() {
	'use strict';

    angular
	    .module('controller.preview',[])
	    .controller('PreviewController', PreviewController);

	    function PreviewController($scope, $state, $stateParams, Posts) {

			var vm = this;

			angular.extend(vm, {
				isDesktop: false,
				previewUrl: decodeURIComponent($stateParams.url),
				back
			});

			function back() {
				$state.go('edit', {'id': $stateParams.id})
			}

	    }
})();
```

**Comment by zachlysobey** [comment]
My guess is that you *somewhere* have a `'use strict'` in an upper scope. Certainly, there are plenty people who use these rules without the problems you are having.

https://jslinterrors.com/unnecessary-use-strict

**Comment by sp90** [comment]
Probably i just turned on jshint after 2 months of work on this product that im doing - but i gotta refactor all the code and make some tests and then find out what is wrong
